builder/FindGlobals.cmake

```diff
@@ -231,6 +230,12 @@ if(UNIX AND MFX_GLIBC)
 endif()
 
 git_describe( git_commit )
-add_definitions( -DMFX_BUILD_INFO=\"${BUILD_INFO}\" )
-add_definitions( -DMFX_API_VERSION=\"${API_VER_MODIF}\" )
-add_definitions( -DMFX_GIT_COMMIT=\"${git_commit}\" )
+
+set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DMFX_BUILD_INFO=\"\\\"${BUILD_INFO}\"\\\"" )
```

Comment:
I think you can again define flags for temporary variable and when add it to the flags. This will be shorter and more readable:
```
set(version_flags -DMFX_BUILD_INFO=\"\\\"${BUILD_INFO}\\\"\"")
...
set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${version_flags}" )
set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${version_flags}" )
```

Reply:
done